lib/module.js

```diff
@@ -79,14 +89,21 @@ async function createFeed(feedOptions, callback) {
   }
 
   const feed = new Feed()
+  let result = ''
```

Comment:
Please avoid mutable variables where possible

Reply:
I'm going to change

Reply:
@manniL changed